rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -810,6 +812,39 @@ def testQuery_ehrConsent(self):
     ps_1 = self.send_get('Participant/%s/Summary' % participant_id_1)
     self.assertEquals('SUBMITTED', ps_1['consentForElectronicHealthRecords'])
 
+  def _submit_dvehr_consent_questionnaire_response(self, participant_id, questionnaire_id,
+                                             dvehr_consent_answer, time=TIME_1):
+    code_answers = []
+    _add_code_answer(code_answers, "DVEHRSharing_AreYouInterested", dvehr_consent_answer)
+    qr = make_questionnaire_response_json(participant_id, questionnaire_id,
+                                          code_answers=code_answers)
+    with FakeClock(time):
+      self.send_post('Participant/%s/QuestionnaireResponse' % participant_id, qr)
+
+  def test_dvehr_consent(self):
+    questionnaire_id = self.create_questionnaire('all_consents_questionnaire.json')
+    participant_1 = self.send_post('Participant', {})
+    participant_id_1 = participant_1['participantId']
+    self.send_consent(participant_id_1)
+    ps_1 = self.send_get('Participant/%s/Summary' % participant_id_1)
+    self.assertEquals('UNSET', ps_1['consentForDvElectronicHealthRecordsSharing'])
+
+    self._submit_dvehr_consent_questionnaire_response(participant_id_1, questionnaire_id,
```

Comment:
We may want a test for submitting the DV EHR consent questionnaire with no answer in it, and confirm it comes back as SUBMITTED_NO_CONSENT.

Reply:
done